WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsActivity.java

```diff
@@ -91,12 +101,21 @@ public void onCreate(Bundle savedInstanceState) {
         } else {
             getIntent().putExtra(KEY_AUTO_REFRESHED, savedInstanceState.getBoolean(KEY_AUTO_REFRESHED));
             getIntent().putExtra(KEY_EMPTY_VIEW_MESSAGE, savedInstanceState.getString(KEY_EMPTY_VIEW_MESSAGE));
-
-            mSelectedCommentId = savedInstanceState.getLong(KEY_SELECTED_COMMENT_ID);
+            mComment = (CommentModel) savedInstanceState.getSerializable(KEY_SELECTED_COMMENT);
         }
     }
 
     @Override
+    public void onStart() {
+        super.onStart();
```

Comment:
My mistake - I read [`public void onCommentChanged(CommentActions.ChangeType changeType)`](https://github.com/wordpress-mobile/WordPress-Android/pull/4864/files/6e2a2d37daf8c2f8f1b659599742c90844435285#diff-fdb14ff3d3a76cacf01c53ff32d68203R328) too quickly, and it looked like a FluxC `OnChanged` event. I agree, disregard.